academic-services/project-academic-knowledge/toc.yml

```diff
@@ -0,0 +1,47 @@
+- name: Project Academic Knowledge documentation
+  href: introduction.md
+- name: Overview
+  expanded: true
+  items:
+  - name: Learn about Project Academic Knowledge
+    href: introduction.md
+- name: Reference
+  expanded: true
+  items:
+  - name: Query expression syntax
+    href: reference-query-expression-syntax.md
+  - name: Entity attributes
+    href: reference-entity-attributes.md
+    items: 
+      - name: Affiliation
+        href: reference-affiliation-entity-attributes.md
+      - name: Author
+        href: reference-author-entity-attributes.md
+      - name: Conference instance
+        href: reference-conference-instance-entity-attributes.md
+      - name: Conference series
+        href: reference-conference-series-entity-attributes.md
+      - name: Field of study
+        href: reference-field-of-study-entity-attributes.md
+      - name: Journal
+        href: reference-journal-entity-attributes.md
+      - name: Paper
+        href: reference-paper-entity-attributes.md
+  - name: REST API
+    expanded: false
+    items:
+      - name: CalcHistogram
+        href: reference-calchistogram-method.md
+      - name: Evaluate
+        href: reference-evaluate-method.md
+      - name: Interpret
+        href: reference-interpret-method.md
+      - name: Similarity
+        href: reference-similarity-method.md
+  - name: Microsoft Research APIs
```

Comment:
Should this be under **Resources**? 
Should we call it "Microsoft Research API Portal" so readers don't confuse it with "REST API"?

Reply:
I was on the fence, but yea I can see that it should prob be there

---
In reply to: [383564519](https://github.com/MicrosoftDocs/microsoft-academic-services/pull/119#discussion_r383564519) [](ancestors = 383564519)